appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -456,6 +490,17 @@
                         (cdr event-info)))
                      events))
 
+         ;; Register the multi events with the Simple event dispatcher
+         (define (register-multi-events events)
+           (define-alias SimpleEventDispatcher
+             <com.google.appinventor.components.runtime.EventDispatcher>)
+           (for-each (lambda (eventName)
+                       ;; Tell the Simple event dispatcher to delegate dispatching of this event to this class
+                       (SimpleEventDispatcher:registerMultiEventForDelegation
+                        (as com.google.appinventor.components.runtime.HandlesEventDispatching (this))
+                        eventName))
```

Comment:
This won't quite work.  What this for-each statement does is execute the lambda procedure over each of the elements of the variable 'events'.  Each of those elements is a pair (called an event-info) comprised of a component name and an event name.  You need to extract the event name from the event-info in order to pass it to registerMultiEventForDelegation.  To do that you use the 'cdr' procedure from Scheme.  If you look at the register-events procedure above you can see how to do that.  Just remove the '(car event-info)' call since registerMultiEventForDelegation only takes an event name as an argument (unlike registerEventForDelegation, which takes a component name and an event name)